tools/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: base
-featured-tool: Open Gazettes
+featured-tool: opengazettes
```

Comment:
Yeah but I changed it so it needs to match the `name` of the tool, not the `title`